helm/.gitignore

```diff
@@ -0,0 +1 @@
+charts/
```

Comment:
Can we move this gitignore changes to the root level file and remove this?

Reply:
Done